domains/middleware/authorization.go

```diff
@@ -274,3 +295,18 @@ func (am *authorizationMiddleware) extAuthorize(ctx context.Context, subj, perm,
 
 	return nil
 }
+
+func (am *authorizationMiddleware) Callout(ctx context.Context, session authn.Session, op string) error {
+	pl := map[string]interface{}{
+		"entity_type": policies.ChannelType,
+		"sender":      session.UserID,
+		"domain":      session.DomainID,
+		"time":        time.Now().String(),
```

Comment:
this fields are different from what billing expects so this request will need to be updated on amdm https://github.com/absmach/amdm/blob/0c2ecf2882fa7704f9a558d43edeb4e01a353aa9/billing/api/http/requests.go#L423